app/controllers/lots_controller.rb

```diff
@@ -41,7 +40,10 @@ def lot_params
       :state,
       :price,
       :category_id,
-      interesting_category_ids: []
+      { interesting_category_ids: [] },
+      :image,
+      { images: [] },
+      :image_cache
```

Comment:
Keep hash params at the end

```suggestion
      :image,
      :image_cache,
      interesting_category_ids: [],
      images: []
```